kubernetes-azure-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,131 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: yaml
+template:
+  description: A YAML document to create a Kubernetes cluster on Azure
+  config:
+    azure-native:location:
+      default: westus2
+      description: The Azure region in which to create the cluster
+    numWorkerNodes:
+      default: 3
+      description: Number of worker nodes in the cluster
+    prefixForDns:
+      default: pulumi
+      description: DNS prefix for the cluster
+    kubernetesVersion:
+      default: 1.24.3
+      description: Kubernetes version to deploy in the cluster
+    nodeVmSize:
+      default: Standard_DS2_v2
+      description: VM size to use for worker nodes in the cluster
+    mgmtGroupId:
+      description: The object ID (from az ad group list) of a group to make cluster admins
+    sshPubKey:
+      description: The public key for SSH access to cluster nodes
+configuration:
+  azure-native:location:
+    type: String
+  numWorkerNodes:
+    type: Number
+    default: 3
+  prefixForDns:
+    type: String
+    default: pulumi
+  kubernetesVersion:
+    type: String
+    default: 1.24.3
+  nodeVmSize:
+    type: String
+    default: Standard_DS2_v2
+  mgmtGroupId:
+    type: String
+  sshPubKey:
+    type: String
+resources:
+  # Create a new resource group
```

Comment:
Partly personal preference, but I tend to like a little whitespace between sections and things, just for a little breathing room. Just makes it a little easier to see where certain things end and others begin.

Reply:
Fixed in 965cc5b (whitespace added to YAML, already present in other languages)